### PR TITLE
Add example for `--show-asr --pass...`, `--show-c` and `--show-llvm` to the blog

### PR DESCRIPTION
**Pull request by Thirumalai-Shaktivel**

### CODE REVIEW

File: content/blog/lpython_mvp.md [hunk 1]
```diff
@@ -45,7 +45,91 @@ for i0 in range(0, length_dim_0):
 
 After applying all the ASR-to-ASR passes, LPython sends the final ASR to the backends selected by the user, via command-line arguments like, `--show-c` (generates C code), `--show-llvm` (generates LLVM code).
 
-<!-- TODO: Add examples of --show-c and --show-llvm command line argument -->
+Add examples of `--show-c` and `--show-llvm`
```
Comment (Smit-create) [inline content/blog/lpython_mvp.md:48]:
```suggestion
One can also see the generated C or LLVM code use the following
```

File: content/blog/lpython_mvp.md [hunk 2]
```diff
@@ -62,7 +146,448 @@ All optimizations are applied via one command-line argument, `--fast`. To select
...
+                                            (IntegerBinOp
+                                                (Var 3 x)
+                                                Add
+                                                (Var 3 i)
+                                                (Integer 4)
+                                                ()
+                                            )
+                                            ()
+                                        )
+                                        ... # Above two assignments are repeated 30 times because of loop_unroll
```
Comment (Thirumalai-Shaktivel) [inline content/blog/lpython_mvp.md:498]: Does this make sense?
Both ASR are 177 and 240 lines

Comment (czgdp1807) [reply content/blog/lpython_mvp.md:498]: You can show the example of inline function calls? That should result in smaller ASR.

Comment (Thirumalai-Shaktivel) [reply content/blog/lpython_mvp.md:498]: Yup! I think that should be small.

Comment (Thirumalai-Shaktivel) [reply content/blog/lpython_mvp.md:498]: I get almost similar lines: 190 lines && 210 lines for the following program. I was not able to simplify further.
```
from lpython import i32

def compute_x() -> i32:
    return (2 * 3) ** 1 + 2

def main():
    x: i32 = compute_x()
    print(x)

main()
```

Comment (Thirumalai-Shaktivel) [reply content/blog/lpython_mvp.md]: We can try unused_function, dead_code_removal

File: content/blog/lpython_mvp.md [hunk 3]
```diff
@@ -62,7 +146,433 @@ All optimizations are applied via one command-line argument, `--fast`. To select
 
 `--pass=inline_function_calls,loop_unroll`
 
-<!-- TODO: Add examples of transformed ASR after applying the above two optimisations -->
+Following is an examples of ASR and transformed ASR after applying the optimisations
```
Comment (czgdp1807) [inline content/blog/lpython_mvp.md:149]:
```suggestion
Following is an examples of ASR and transformed ASR after applying the optimisations

```

**Review by czgdp1807 — APPROVED** [review]
Overall this looks good to me.

**Comment by Thirumalai-Shaktivel** [comment]
Cool, thanks for the review!

**Comment by Shaikh-Ubaid** [comment]
Is this PR ready?

**Comment by Thirumalai-Shaktivel** [comment]
Yup!

**Comment by Thirumalai-Shaktivel** [comment]
Please squash merge both the PRs! Thank you!

**Comment by Shaikh-Ubaid** [comment]
>Please squash merge both PRs! Thank you!

I was about to ask you in this PR. Thank you!

**Pull request merged**